File: art_md_sim_02_1/PrintSummary.jl
function PrintSummary()
  @printf("(%5d,%18.10f) [%18.10f] (%18.10f,%18.10f) (%18.10f,%18.10f) (%18.10f,%18.10f)\n",
     stepCount, timeNow, (vSum.x + vSum.y)/nMol,
     totEnergy.sum, totEnergy.sum2, kinEnergy.sum, kinEnergy.sum2,
     pressure.sum, pressure.sum2 )
end

File: ART_MD/PROG_02_1_C/PrintSummary.c
void PrintSummary (FILE *fp)
{
  fprintf(fp,
     "%5d %8.4f %7.4f %7.4f %7.4f %7.4f %7.4f %7.4f %7.4f\n",
     stepCount, timeNow, VCSum (vSum) / nMol, PropEst (totEnergy),
     PropEst (kinEnergy), PropEst (pressure));
  fflush (fp);
}
